gravitee-apim-portal-webui-next/src/components/card/card.component.scss

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+.api-card {
+  display: flex;
+  width: calc(33.33% - 20px);
+
+  --mdc-elevated-card-container-shape: 12px;
+
+  border: 1px solid #cac4d0;
+
+  &__header {
+    display: flex;
+    align-items: center;
+    gap: 16px;
+  }
+
+  &__header-logo {
+    border-radius: 50%;
+  }
+
+  &_header-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    gap: 4px;
+  }
+
+  &__header-content-title {
+    font-size: 16px;
+    font-weight: 500;
+  }
+
+  &__header-content-version {
+    font-size: 14px;
+    font-weight: 400;
+    letter-spacing: 0.25px;
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    gap: 32px;
+  }
+
+  &__content-description {
+    color: #49454f;
```

Comment:
Instead of using a specific color, put the description in a `<p>` and the color should be adjusted correctly

Reply:
Done